Source/WebCore/page/csp/ContentSecurityPolicy.cpp

```diff
@@ -955,6 +949,16 @@ void ContentSecurityPolicy::reportViolation(const String& effectiveViolatedDirec
     if (usesReportTo)
         httpStatusCode = m_httpStatusCode;
 
+    // WPT results indicate an explicit port number is needed in the reporting results. WebKit does not provide an
+    // explicit port in the parsed URL when that port is the default port for the protocol. Therefore we build
+    // a reporting string with this information:
```

Comment:
I should remove this: Chrome also elides the port number when it matches the canonical port for the protocol.